lib/write-file.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var path = require('path');
+var fs = require('fs');
+
+function uniqueFile(file) {
+  var testPath = file;
+  var counter = null;
+  while (fs.existsSync(testPath + '.png')) {
+    counter = (counter || 0) + 1;
+    testPath = file + counter;
+  }
+
+  return testPath + '.png';
+}
+
+function getFile(directory, title) {
```

Comment:
`generateFilename`? "slug" implies dashes for me, so I feel like it's too specific.